.github/workflows/tests_coverage_deploy.yml

```
@@ -50,8 +58,34 @@ jobs:
         pip install pytest pytest-asyncio httpx pytest-cov
         pip install -r backend/requirements.txt
 
+    - name: Set up Docker Compose
+      uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)